<ui version="4.0" >
 <class>KMahjonggTilesetSelector</class>
 <widget class="QWidget" name="KMahjonggTilesetSelector" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>446</height>
   </rect>
  </property>
  <property name="maximumSize" >
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QListWidget" name="tilesetList" >
       <property name="minimumSize" >
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2" >
       <property name="title" >
        <string>Preview</string>
       </property>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QLabel" name="tilesetPreview" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>5</hsizetype>
            <vsizetype>5</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>120</width>
            <height>160</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Properties</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelAuthor" >
        <property name="text" >
         <string>Author:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="tilesetAuthor"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelContact" >
        <property name="text" >
         <string>Contact:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="tilesetContact"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelDescription" >
        <property name="text" >
         <string>Description:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="tilesetDescription"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" >
        <property name="text" >
         <string>Version:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="tilesetVersion"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" >
        <property name="text" >
         <string>Website:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="tilesetWebsite">
        <property name="openExternalLinks" >
         <bool>true:</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" >
        <property name="text" >
         <string>Copyright:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="tilesetCopyright"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" >
        <property name="text" >
         <string>License:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="tilesetLicense"/>   
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="kcfg_TileSet" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="frame" >
      <bool>false</bool>
     </property>
     <property name="readOnly" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
